MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -16,15 +16,22 @@ public enum VisualInstructionComponentType: Int, CustomStringConvertible {
     /**
      The component bears an exit number or the name of a place or street.
      */
-    case destination
+    case text
+    
+    /**
+     Component contains an icon that should be rendered.
+     */
+    case icon
```

Comment:
`image` tends to be used more often than `icon` in Cocoa frameworks. `image` would also highlight the fact that these components have non-nil `image` properties.